Description: Fix 'indifferenctly' typo, replace with 'indifferently'.
Author: Bas Couwenberg <sebastic@xs4all.nl>
Last-Update: 2013-09-30
--- a/spatialite_osm_map.c
+++ b/spatialite_osm_map.c
@@ -2548,7 +2548,7 @@ do_help ()
     fprintf (stderr,
 	     "                 both OSM-XML (*.osm) and OSM-ProtoBuf\n");
     fprintf (stderr,
-	     "                 (*.osm.pbf) are indifferenctly supported.\n\n");
+	     "                 (*.osm.pbf) are indifferently supported.\n\n");
     fprintf (stderr,
 	     "-d or --db-path  pathname       the SpatiaLite DB path\n\n");
     fprintf (stderr, "you can specify the following options as well\n");
--- a/spatialite_osm_net.c
+++ b/spatialite_osm_net.c
@@ -2255,7 +2255,7 @@ do_help ()
     fprintf (stderr,
 	     "                 both OSM-XML (*.osm) and OSM-ProtoBuf\n");
     fprintf (stderr,
-	     "                 (*.osm.pbf) are indifferenctly supported.\n\n");
+	     "                 (*.osm.pbf) are indifferently supported.\n\n");
     fprintf (stderr,
 	     "-d or --db-path  pathname       the SpatiaLite DB path\n");
     fprintf (stderr,
--- a/spatialite_osm_raw.c
+++ b/spatialite_osm_raw.c
@@ -947,7 +947,7 @@ do_help ()
     fprintf (stderr,
 	     "                 both OSM-XML (*.osm) and OSM-ProtoBuf\n");
     fprintf (stderr,
-	     "                 (*.osm.pbf) are indifferenctly supported.\n\n");
+	     "                 (*.osm.pbf) are indifferently supported.\n\n");
     fprintf (stderr,
 	     "-d or --db-path  pathname       the SpatiaLite DB path\n\n");
     fprintf (stderr, "you can specify the following options as well\n");
